docs/intro/setup.rst

```diff
@@ -72,29 +72,52 @@ Here we give line-by-line instructions to demonstrate the general workflow.
 .. code-block:: python
 
    import hls4ml
+   import tensorflow as tf
+   from tensorflow.keras.layers import Dense
 
-   # Fetch a keras model from our example repository
-   # This will download our example model to your working directory and return an example configuration file
-   config = hls4ml.utils.fetch_example_model('KERAS_3layer.json')
+   # Construct a basic, untrained keras model
+   model = tf.keras.models.Sequential()
+   model.add(Dense(64, input_shape=(16,), name='Dense', kernel_initializer='lecun_uniform', kernel_regularizer=None))
+   model.add(Activation(activation='elu', name='Activation'))
+   model.add(Dense(32, name='Dense2', kernel_initializer='lecun_uniform', kernel_regularizer=None))
+   model.add(Activation(activation='elu', name='Activation2'))
 
-   # You can print it to see some default parameters
+   # Generate an hls configuration from the keras model
+   config = hls4ml.utils.config_from_keras_model(model)
+
+   # You can print the config to see some default parameters
    print(config)
 
-   # Convert it to a hls project
-   hls_model = hls4ml.converters.keras_to_hls(config)
+   # Convert the model to an hls project using the config
+   hls_model = hls4ml.converters.convert_from_keras_model(
+      model=model,
+      hls_config=config,
+      backend='Vitis'
+   )
+
+Once converted to an HLS project, you can connect the project into the Python runtime and use it to run predictions on a numpy array:
+
+.. code-block:: python
```

Comment:
For some reason this is not appearing in the docs when I do `make html` locally. I see in the messages:
```
/Users/jmitrevs/work/hls4ml/docs/intro/setup.rst:100: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 5 supplied.

.. code-block:: python
   import numpy as np

   # Compile the hls project and link it into the Python runtime
   hls_model.compile()

   # Generate random input data
   X_input = np.random.rand(100, 16)

   # Run the model on the input data
   hls_prediction = hls_model.predict(X_input) [docutils]
```